server/sonar-server/src/main/java/org/sonar/server/qualitygate/ws/QualityGatesWs.java

```diff
@@ -54,6 +56,20 @@ public void define(Context context) {
       action.define(controller);
     }
 
+    // unset_default is no more authorized
+    controller.createAction("unset_default")
+      .setDescription("Unset the default quality gate is no more authorized. A quality gate is now required.<br>" +
```

Comment:
I don't understand "A quality gate is now required". You mean "At least one default quality gate is now always required" ?

Reply:
So it's not clear enough !
This WS has the ability to unset the default, as a consequence of there is no more default quality gate and it's no more possible.